### ISSUE TITLE
Accessibility: Related links that are not grouped in a list (Failure WCAG 1.3.1)

### ISSUE THREAD

**Issue by RababGomaa** [canada-ca/design-system-systeme-conception#92]
[Institutional landing page - currently in beta](https://design.canada.ca/coded-layout/institutional_landing_page_guidance.html)
Mandate , Organizational structure, Transparency, .. : Example of related links that are not grouped in a list  

Suggested solution: 
Use `<ul>` to group the links

**Related WCAG technique:** 
[H48: Using ol, ul and dl for lists or groups of links](https://www.w3.org/TR/WCAG20-TECHS/H48.html)

**Comment by jmealing** [comment]
These links have now been converted to a list and the changes are live - thank you @RubyDo
